.gitmodules

```diff
@@ -94,4 +94,7 @@
 [submodule "externals/pugixml"]
 	path = externals/pugixml
 	url = https://github.com/zeux/pugixml.git
-	shallow = true
\ No newline at end of file
+	shallow = true
+[submodule "externals/discord-rpc"]
+	path = externals/discord-rpc
+	url = https://github.com/delledev/discord-rpc
```

Comment:
You should use https://github.com/shadps4-emu/ext-discord-rpc

Reply:
Thank you for your input. I've changed it.